pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -921,6 +921,7 @@
        "[{'type': 'UncategorizedText',\n",
        "  'element_id': 'db1ca22813f01feda8759ff04a844e56',\n",
        "  'metadata': {'filename': 'family-day.eml',\n",
+       "   'file_directory': '/Users/shreyanid/Documents/all-unstructured/unstructured-api/sample-docs',\n",
```

Comment:
Looks like this field has been in the metadata for a while - did something just change for it to show here? This will probably get us into trouble with `make check-notebooks`.

Reply:
That's exactly what's happening :') Not sure what just changed, [this commit](https://github.com/Unstructured-IO/unstructured/commit/f8a7ae2953ff4dfb06cc42ca9fc62a36a053fab0) was thrown out as a possibility. But it isn't happening on the current main branch of api, so that means some package bump (likely unstructured) caused the change. Would appreciate some help debugging where the change came from, and why the difference in file directory is not happening locally but happening in CI.

Reply:
In [this](https://github.com/Unstructured-IO/unstructured/compare/0.9.1...0.9.2) release comparison it doesn't seem like anything recently changed related to the file_directory field

Reply:
Hmm, that's a fun one! I have some cycles now to take a look as well.

Reply:
Ah, I think it is that commit. [Here](https://github.com/Unstructured-IO/unstructured/blob/79a1eb86837ccfd84eb265985d6981d9ef9fe965/unstructured/documents/elements.py#L173) it looks like `file_directory` will be set whenever a filename is present, and that commit will have us sending `metadata_filename` all the time. All the different filename params confuse me, but tldr is that we should remove that field like the ones [over here](https://github.com/Unstructured-IO/unstructured-api/blob/c9b74d43e716fb36eb2c07b62ae3eb237feb4e15/prepline_general/api/general.py#L367).